internal/config/config_test.go

```diff
@@ -150,6 +151,29 @@ func addContext() {
 	})
 }
 
+func TestAddContextWithEmptyUser(t *testing.T) {
```

Comment:
>c TestAddContextWithEmptyUser [](http://example.com/codeflow?start=3&length=29)

how do we know the content of the file is correct?

Reply:
We don't.  We are presuming it is like a .mdf file right now, i.e. it hasn't been futzed with.  Do you think we should harden sqlcmd to work with sqlconfig files that have been manually altered?

Reply:
Added discussion:

[https://github.com/microsoft/go-sqlcmd/discussions/264](https://github.com/microsoft/go-sqlcmd/discussions/264)

Reply:
that's not what I meant. The existence of the Test method implies that there's a need to validate that the AddContext method is putting the correct value into the yaml file. 
I don't see any validation that it's writing the correct fields.
It's just validating that there's no panic.

Reply:
Agree, I've added a check to ensure user is nil on retrieval

 context := GetContext("context")
 assert.Nil(t, context.User)